src/Layout/UI/FileInput.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useRef, useState } from "react";
+import "./FileInput.css";
+import xbutton from "../../images/icons/ic_X.svg";
+
+function FileInput({ name, value, onChange }) {
+  const [preview, setPreview] = useState();
+  const inputRef = useRef();
+
+  const handleChange = (e) => {
+    const nextValue = e.target.files[0];
+    onChange(name, nextValue);
+  };
+
+  const handleClearClick = () => {
+    const inputNode = inputRef.current;
+    if (!inputNode) return;
+
+    inputNode.value = "";
+    onChange(name, null);
+  };
+
+  useEffect(() => {
+    if (!value) return;
+
+    const nextPreview = URL.createObjectURL(value);
+    setPreview(nextPreview);
+
+    return () => {
+      setPreview();
+      URL.revokeObjectURL(nextPreview);
+    };
+  }, [value]);
+
+  return (
+    <div className="fileInput-container">
+      <img src={preview} alt="" className="inputImage" />
+      <input
+        id="imgFile"
+        type="file"
+        accept="image/png, image/jpeg"
+        onChange={handleChange}
+        ref={inputRef}
+      />
+      {value && (
+        <button onClick={handleClearClick} className="button">
+          <img src={xbutton} alt="취소버튼" />
+        </button>
+      )}
```

Comment:
`preview`가 undefined 또는 nullable일때  이미지 태그를 계속 렌더링 시켜주어서 이미지를 없애더라도 깨진 이미지라고 렌더링이 되었습니다. 
조건부 렌더링을 이용해 이미지가 없다면 이미지 태그 자체를 보여주지 않도록 해주시면 됩니다.

```suggestion
<input
        id="imgFile"
        type="file"
        accept="image/png, image/jpeg"
        onChange={handleChange}
        ref={inputRef}
      />
      {value && (
        <>
          <button onClick={handleClearClick} className="button">
            <img src={xbutton} alt="취소버튼" />
          </button>
          <img src={preview} alt="" className="inputImage" />
        </>
      )}```